transcom/mymove
# MB-7641 Add submitted at filter

## Description

This PR filters the list by the submitted at and adds an index to the moves table for the submitted at column.

## Setup

`make server_test`

To verify in the UI complete the following steps:
- start the server: `make server_run`
- start the client: `make client_run`
- navigate to `http://officelocal:3000/counseling/queue` as the TOO
- in the submitted at, filter the column
![Screen Shot 2021-04-15 at 3 11 47 PM](https://user-images.githubusercontent.com/56279459/114950006-aa56c780-9dfe-11eb-98a5-61bbfc6b68ad.png)


## Code Review Verification Steps

* Any new migrations/schema changes:

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-7641) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._
